test/adapter/json_api/linked_test.rb

```diff
@@ -44,8 +45,8 @@ def test_include_multiple_posts_and_linked
             @second_comment.post = @first_post
             @second_comment.author = nil
             assert_equal([
-                           { title: "Hello!!", body: "Hello, world!!", id: "1", links: { comments: ['1', '2'], author: "1" } },
-                           { title: "New Post", body: "Body", id: "2", links: { comments: [], :author => "2" } }
+                           { :id=>"1", :title=>"Hello!!", :body=>"Hello, world!!", :links=>{:comments=>["1", "2"], :blog=>"999", :author=>"1"} },
+                           { :id=>"2", :title=>"New Post", :body=>"Body", :links=>{:comments=>[], :blog=>"999", :author=>"2"} }
```

Comment:
Can we get rid of the hash-rocket version just for consistency? We have it differently everywhere. Also, it feels so Ruby 1.9 :grin: